### PR TITLE
Fixed pl.BlockSpec argument ordering in the Pallas TPU matmul tutorial

### PR DESCRIPTION
**Pull request by superbobry**

### CODE REVIEW

File: docs/pallas/tpu/matmul.ipynb [hunk 1]
```diff
@@ -674,7 +674,7 @@
     "\n",
     "### Fused right-hand-side transpose\n",
     "\n",
-    "A common first thing to do is to fuse a transpose. What do we mean by that? Suppose we wanted to compute `x @ y.T` instead of `x @ y`. Naively we could first compute `y.T` and then pass it into our efficient matrix multiply kernel. However, the operation `y.T` is not free on its own -- it involves copying `O(n^2)` data. Ideally, we could compute the transpose *while* doing the matrix multiply in just one kernel, i.e. \"fusing\" it with the matmul. \n",
```
Comment (gnecula) [inline docs/pallas/tpu/matmul.ipynb:677]: Maybe break this into multiple lines?

File: docs/pallas/tpu/matmul.md [hunk 2]
```diff
@@ -428,7 +428,7 @@ Now that we have a basic matrix multiplication kernel, we can now try fusing ope
 
 ### Fused right-hand-side transpose
 
-A common first thing to do is to fuse a transpose. What do we mean by that? Suppose we wanted to compute `x @ y.T` instead of `x @ y`. Naively we could first compute `y.T` and then pass it into our efficient matrix multiply kernel. However, the operation `y.T` is not free on its own -- it involves copying `O(n^2)` data. Ideally, we could compute the transpose *while* doing the matrix multiply in just one kernel, i.e. "fusing" it with the matmul.
```
Comment (gnecula) [inline docs/pallas/tpu/matmul.md:431]: And this can be multiple lines also

**Review by gnecula — APPROVED** [review]

File: docs/pallas/tpu/matmul.ipynb (hunk 1 above)
Comment (superbobry) [reply docs/pallas/tpu/matmul.ipynb:677]: All paragraphs in the tutorial are single lines AFAICT. So, if we want to break them into multiple lines, we probably need to do that to all of them (and in a separate PR?).

File: docs/pallas/tpu/matmul.md (hunk 2 above)
Comment (superbobry) [reply docs/pallas/tpu/matmul.md:431]: Ack.

**Pull request merged**